wgkex/config/config.py

```diff
@@ -30,11 +36,13 @@ class MQTT:
         tls: If TLS is used or not.
         broker_port: The port for MQTT to connect on.
         keepalive: The keepalive in seconds to use.
+        key_prefix: The prefix to pre-pend to a given domain, interface.
```

Comment:
Should we rename this to `domain_prefix` as it's related to the domains and not the `key`?

Reply:
Agree, thank you for pointing this out.